test/node/utf8_tests.js

```diff
@@ -0,0 +1,343 @@
+'use strict';
+
+const { Buffer } = require('buffer');
+const BSON = require('../register-bson');
+const { isNode6, isBrowser } = require('./tools/utils');
+const BSONError = BSON.BSONError;
+
+describe('UTF8 validation', function () {
+  // Test both browser shims and node which have different replacement mechanisms
+  const replacementChar = isNode6() || isBrowser() ? '���' : '�';
```

Comment:
```suggestion
  const replacementChar = isNode6() || isBrowser() ? '\u{FFFD}\u{FFFD}\u{FFFD}' : '\u{FFFD}';
```

I think this just calls out the code point better for the next reader, just a suggestion.